validator/testdata/amp4ads_feature_tests/amp_pixel_ar.html

```diff
@@ -0,0 +1,17 @@
+<!--
+  Test Description:
+  Verify amp-pixel with Attribution Reporting API is valid.
+-->
+<!doctype html>
+<html ⚡4ads>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width,minimum-scale=1">
+  <style amp4ads-boilerplate>body{visibility:hidden}</style>
+  <script async src="https://cdn.ampproject.org/amp4ads-v0.js"></script>
+</head>
+<body>
+  Hello, world.
+  <amp-pixel src="https://pixel.me?yes=please" attributionsrc=""></amp-pixel>
```

Comment:
can we put another `<amp-pixel>` with https url

Reply:
Yep, added.